nebula_common/include/nebula_common/continental/continental_srr520.hpp

```diff
@@ -0,0 +1,296 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <nebula_common/nebula_common.hpp>
+#include <nebula_common/nebula_status.hpp>
```

Comment:
`#include ""`

Reply:
Addressed in [6819c28](https://github.com/tier4/nebula/pull/124/commits/6819c282bb6ae1638962f665bb9e7507f935d373)